web3/index.js

```diff
@@ -89,3 +95,17 @@ function loadScripts () {
   });
 }
 
+function configureApi (config) {
+  const { DAPPS } = config;
+
+  const dappsInterface = DAPPS.split(':')[0];
+  const dappsPort = DAPPS.split(':')[1];
+
+  if (dappsInterface) {
+    window.secureApi._dappsInterface = dappsInterface;
```

Comment:
A bit hacky to modify private fields. Can we expose this as a method in new versions of Parity?